docs/source/guides/installation_guide.md

```diff
@@ -55,12 +55,12 @@ Here is the recommended procedure for setting up `nucypher` in this fashion:
     Activate the newly created virtual environment:
     
     ```bash
-    $ source /your/path/nucypher-venv
+    $ source /your/path/nucypher-venv/bin/activate
     ...
     $(nucypher-venv)
     ```
 
-    ``` note:: Successful virtualenv activation is indicated by '(nucypher-venv)$' prepending your console's prompt
+    ``` note:: Successful virtualenv activation is indicated by `(nucypher-venv)$` prepending your console's prompt
```

Comment:
```suggestion
    ``` note:: Successful virtualenv activation is indicated by `(nucypher-venv)$` prepended to your console's prompt
```

Reply:
Thanks! I'll correct this in #1232

Reply:
👍